exasol/ds/sandbox/runtime/ansible/dss_docker_playbook.yml

```diff
@@ -0,0 +1,22 @@
+- name: Prepare environment
+  hosts: localhost
+  gather_facts: false
+  vars:
+      ansible_python_interpreter: python3
+  tasks:
+    - name: Add docker container to inventory
+      add_host:
+        name: "{{docker_container}}"
+        ansible_connection: docker
+
+- name: Setup DSS Docker Container
+  hosts: ds-sandbox-docker # cannot use variable here
```

Comment:
Ok, this gets an interesting name, because if we need to fix this name, there are no concurrent runs allowed. Let's discuss this later.